content/docs/user-guide/how-to/setup-google-drive-remote.md

```diff
@@ -248,12 +251,6 @@ $ dvc remote modify myremote gdrive_acknowledge_abuse true
 </admon>
 > Please note our [Privacy Policy (Google APIs)](/doc/user-guide/privacy).
```

Comment:
Unrelated, but this is broken. Where is it supposed to link to/do we need it?

Reply:
It's not broken 🙂 just not linked from nav: 

https://dvc.org/doc/user-guide/privacy

Reply:
```suggestion
> Please note our [Privacy Policy (Google APIs)](https://dvc.org/doc/user-guide/privacy).
```

So how about this?

Reply:
Ooooh I see what you mean. It does not render correctly it seems. Interesting... OK committing, thanks.